less/index.less

```diff
@@ -52,8 +52,126 @@ table {
 * {
     box-sizing: border-box;
 }
+// Variables
+@background-color: @Gallery;
+@Gallery: #F0F0F0;
+
+@h2-font-size: 2.5rem;
+@p-font-size: 1rem;
 
 html, body {
+	background: @Gallery;
     height: 100%;
 }
+.container-home {
+	max-width: 1200px;
+	width: 100%;
+	margin: 0 auto;
+
+	.header {
+		margin: 1% 0 2% 0;
+
+		h1 {
+		display: flex;
+		justify-content: center;
+		font-size: 3rem;
+	
+		}
+		.nav {
+		display: flex;
+		flex-direction: row;
+		width: 100%;
+		justify-content: space-between;
+		font-size: 1.2rem;
+		
+		a {
+			color: black;
+			text-decoration: none;
+			
+		}	
+			a:hover {
```

Comment:
We didn't go over this until today, but going forward, be sure to use the `&:hover` syntax as to not repeat selectors unnecessarily.